test/integration/other/encryption.js

```diff
@@ -376,7 +376,7 @@ describe('managed encryption', () => {
             where action='submission.attachment.update' and processed is not null and failures = 0`)
             .then((count) => { count.should.equal(4); })))));
 
-    it('should decrypt client audit log attachments', testService((service) =>
+    it('should decrypt client audit log attachments (2)', testService((service) =>
```

Comment:
This test is more accurately about decrypting encrypted client audit attachments during export so I think the name can stay the same.